<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Music Player</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>12</number>
    </property>

    <!-- ===== COVER CONTAINER ===== -->
    <item>
     <widget class="QFrame" name="coverContainer">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <layout class="QVBoxLayout" name="coverLayout">
       <item alignment="Qt::AlignCenter">
        <widget class="QLabel" name="coverLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <!-- ===== TRACK ===== -->
    <item>
     <widget class="QLabel" name="trackTitleLabel">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QLabel" name="artistLabel">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>

    <!-- ===== CONTROL PANEL ===== -->
    <item>
     <widget class="QFrame" name="controlPanel">
      <property name="minimumHeight">
       <number>120</number>
      </property>
      <property name="maximumHeight">
       <number>120</number>
      </property>
      <layout class="QHBoxLayout" name="controlLayout">
       <property name="spacing">
        <number>24</number>
       </property>

       <item>
        <widget class="QPushButton" name="prevBtn">
         <property name="minimumHeight">
          <number>60</number>
         </property>
        </widget>
       </item>

       <item>
        <spacer name="spacer1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>

       <item>
        <widget class="QPushButton" name="playBtn">
         <property name="minimumHeight">
          <number>60</number>
         </property>
        </widget>
       </item>

       <item>
        <spacer name="spacer2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
        </spacer>
       </item>

       <item>
        <widget class="QPushButton" name="nextBtn">
         <property name="minimumHeight">
          <number>60</number>
         </property>
        </widget>
       </item>

      </layout>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
